programs/drift/src/state/order_params.rs

```diff
@@ -402,31 +429,27 @@ impl OrderParams {
                     .last_oracle_price_twap_5min,
             )?;
 
+        let frac_of_long_spread_in_price: i64 = perp_market
+            .amm
+            .long_spread
+            .cast::<i64>()?
+            .safe_mul(mark_twap_slow)?
+            .safe_div(PRICE_PRECISION_I64 * 10)?;
+
+        let frac_of_short_spread_in_price: i64 = perp_market
+            .amm
+            .short_spread
+            .cast::<i64>()?
+            .safe_mul(mark_twap_slow)?
+            .safe_div(PRICE_PRECISION_I64 * 10)?;
+
         let baseline_start_price_offset = match direction {
-            PositionDirection::Long => {
-                let frac_of_spread_in_price: i64 = perp_market
-                    .amm
-                    .long_spread
-                    .cast::<i64>()?
-                    .safe_mul(mark_twap_slow)?
-                    .safe_div(PRICE_PRECISION_I64 * 10)?;
-
-                baseline_start_price_offset_slow
-                    .max(baseline_start_price_offset_fast)
-                    .safe_add(frac_of_spread_in_price)?
-            }
-            PositionDirection::Short => {
-                let frac_of_spread_in_price: i64 = perp_market
-                    .amm
-                    .short_spread
-                    .cast::<i64>()?
-                    .safe_mul(mark_twap_slow)?
-                    .safe_div(PRICE_PRECISION_I64 * 10)?;
-
-                baseline_start_price_offset_slow
-                    .min(baseline_start_price_offset_fast)
-                    .safe_sub(frac_of_spread_in_price)?
-            }
+            PositionDirection::Long => baseline_start_price_offset_slow
+                .safe_add(frac_of_long_spread_in_price)?
+                .min(baseline_start_price_offset_fast.safe_sub(frac_of_short_spread_in_price)?),
```

Comment:
so before we were doing max, which gives the more aggressive, now we're doing min, which is less aggresswive?

Reply:
correct, doing min for long is less aggressive